Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fabiorphp/golang-glide:alpine as glide
+RUN apk update
+RUN apk add git
+WORKDIR /go/src/github.com/jirwin/burrow_exporter
+COPY . /go/src/github.com/jirwin/burrow_exporter
+RUN glide install
+RUN go build burrow-exporter.go
+
+FROM alpine
+COPY --from=glide /go/src/github.com/jirwin/burrow_exporter/burrow-exporter .
+CMD ./burrow-exporter --burrow-addr $BURROW_ADDR --metrics-addr $METRICS_ADDR --interval $INTERVAL
```

Comment:
`$METRICS_ADDR` and `$INTERVAL` should have defaults `ENV METRICS_ADDR=0.0.0.0:8080 INTERVAL=30`